main.nf

```diff
@@ -3136,15 +3136,33 @@ process kraken {
 
   output:
   file "*.kraken.out" into ch_kraken_out
-  tuple prefix, path("*.kreport") into ch_kraken_report, ch_kraken_for_multiqc
+  tuple prefix, path("*.kraken2_report") into ch_kraken_report, ch_kraken_report_backward_compatibility
```

Comment:
@maxibor would waiting until MQC 1.10 is release remove the need for this?

Reply:
Pretty sure it would ==> rather would try pushing this for next release in MQC 1.10 :-)